functions/Export-DbaUser.ps1

```diff
@@ -489,12 +492,12 @@ function Export-DbaUser {
                         $sql = "$useDatabase $outsql"
                     } else {
                         if ($useDatabase) {
-                            $sql = "$useDatabase`r`nGO`r`n" + ($outsql -join "`r`nGO`r`n")
+                            $sql = "$useDatabase${eol}GO$eol" + ($outsql -join "${eol}GO$eol")
                         } else {
-                            $sql = $outsql -join "`r`nGO`r`n"
+                            $sql = $outsql -join "${eol}GO$eol"
```

Comment:
Thank you so much for this PR! What is `${eol}`? I'm not familiar with this syntax

Reply:
In `$eolGO` the name of the variable would include the GO. The `${}` is a bit shorter than the alternative `$($eol)`. Should I change that to `$($eol)`?

Reply:
ahh! Interesting. yes, $($eol)) would be the standard way we handle these, thank you.

Reply:
Will do the change later today...

Reply:
And changed...